src/data/finset/basic.lean

```diff
@@ -2258,6 +2260,15 @@ begin
   { rw [erase_eq_of_not_mem h], apply nat.sub_le }
 end
 
+theorem card_erase [decidable_eq α] {a : α} {s : finset α} :
+  card (erase s a) = if a ∈ s then pred (card s) else card s
```

Comment:
Can you prove this for `multiset` too?

Reply:
Also perhaps worth having a reference to the versions that avoid introducing an `ite`:
```suggestion
/-- If `a ∈ s` is known, see also `finset.card_erase_of_mem` and `finset.erase_eq_of_not_mem`. -/
theorem card_erase [decidable_eq α] {a : α} {s : finset α} :
  card (erase s a) = if a ∈ s then pred (card s) else card s
```

Reply:
Done!